source/_docs/unsupported-modules-plugins.md

```diff
@@ -445,6 +445,19 @@ The list of [WordPress roles and capabilities](https://codex.wordpress.org/Roles
 * `edit_plugins`
 * `edit_themes`
 
+## Paid Plugins or Licenses
+
+In general, licensed or paid plugins, modules or themes are allowed for use in Pantheon. For their use, consider platform constrants or conflicts that will arise due to the nature of the platform. Licensing issues are for Customers to Resolve, as per the terms of their plugin or themes.
```

Comment:
"consider platform constrants or conflicts that will arise due to the nature of the platform."

This is ambiguous to the point of not being helpfuil. What constraints specifically affect licensed plugins?